src/System.Text.Json/tests/Serialization/Value.ReadTests.GenericCollections.cs

```diff
@@ -1143,7 +1144,15 @@ public static void ReadSimpleTestClass_GenericWrappers_NoAddMethod_Throws()
         [Fact]
         public static void ReadPrimitiveStringCollection_Throws()
         {
-            Assert.Throws<InvalidCastException>(() => JsonSerializer.Deserialize<StringCollection>(@"[""1"", ""2""]"));
+            Assert.Throws<NotSupportedException>(() => JsonSerializer.Deserialize<StringCollection>(@"[""1"", ""2""]"));
```

Comment:
Can we add `StringValues` based on the original issue?

Reply:
You want to copy the entire `StringValues` type into corefx for this test?

Reply:
Wouldn't a "ProjectReference Include" to System.Collections.Specialized in the test project work?

Reply:
`StringValues` is an [ASP.NET Core type](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.primitives.stringvalues?view=aspnetcore-2.2) living in [aspnet/Extensions](https://github.com/aspnet/Extensions), AFAIK that's not possible.

Reply:
I see. nevermind :)